alerting/grafana_alertmanager.go

```diff
@@ -192,15 +194,15 @@ func newAlertmanager(ctx context.Context, orgID int64, cfg *setting.Cfg, store A
 
 	am.wg.Add(1)
 	go func() {
-		am.silences.Maintenance(silenceMaintenanceInterval, silencesFilePath, am.stopc, func() (int64, error) {
+		am.silences.Maintenance(config.Silences.MaintenanceFrequency(), config.Silences.Filepath(), am.stopc, func() (int64, error) {
 			// Delete silences older than the retention period.
 			if _, err := am.silences.GC(); err != nil {
 				am.logger.Error("silence garbage collection", "err", err)
 				// Don't return here - we need to snapshot our state first.
 			}
 
 			// Snapshot our silences to the Grafana KV store
```

Comment:
I think the comment might need an update too as it's not the Grafana KV store exclusively.

Reply:
Noted in #8